io-engine/src/bin/io-engine-client/v1/replica_cli.rs

```diff
@@ -331,6 +336,8 @@ async fn replica_list(
                     "URI",
                     "IS_SNAPSHOT",
                     "IS_CLONE",
+                    "SNAP_ANCSTR_SIZE",
```

Comment:
`SNAP_ANCESTOR_SIZE` Since only 2 letters maybe we can add?

Reply:
It was just kept to save space in the console to add new fields. :)
Anyways, I have changed it. In future if any new fields comes, then We will realing all the fields accordingly.